django/db/models/expressions.py

```diff
@@ -420,6 +421,18 @@ class Expression(BaseExpression, Combinable):
 
 class CombinedExpression(SQLiteNumericMixin, Expression):
 
+    _connector_output_fields = {
+        (connector, lhs, rhs): output_field
+        for connector in (Combinable.ADD, Combinable.SUB, Combinable.MUL, Combinable.DIV)
+        for lhs, rhs, output_field in (
+            (fields.IntegerField, fields.DecimalField, fields.DecimalField),
+            (fields.DecimalField, fields.IntegerField, fields.DecimalField),
+            (fields.IntegerField, fields.FloatField, fields.FloatField),
+            (fields.FloatField, fields.IntegerField, fields.FloatField),
+            (fields.FloatField, fields.IntegerField, fields.FloatField),
```

Comment:
This line is a duplicate of the one above?

Reply:
yep, I'll drop the line.